File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/03_compito_java_20150216/Libro.py
from Prodotto import Prodotto


class Libro(Prodotto):
    def __init__(self, tipo, codice, titolo, autore, prezzo, isbn):
        super().__init__(tipo, codice, titolo, autore, prezzo)
        self._isbn = isbn

    def __str__(self):
        return super().__str__() + "\t" + self._isbn + "\t-"
    

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/03_compito_java_2015_02_16/Libro.java
import java.util.*;
import java.io.*;

public class Libro extends Prodotto {
    private String isbn;

    public Libro(int codice, String titolo, String autori, int prezzo, String isbn) {
        super(codice, titolo, autori, prezzo);
        this.isbn = isbn;
    }

    public String toString() {
        return "libro\t" + codice + "\t" + titolo + "\t" + autori + "\t" + prezzo + "\t" + isbn + "\t-";
    }
}